### PR TITLE
Add Galactic type handling

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

**Review by mamerlan — DISMISSED** [review]

File: _docs/tutorials/advanced/handling_type_memory/index.md [hunk 1]
```diff
@@ -191,3 +201,122 @@ for(int32_t i = 0; i < 3; i++){
   mymsg.multiheaders.size++;
 }
 ```
+
+# micro-ROS Galactic
+
+Due to the inclusion of [`rosidl_typesupport_introspection_c`](https://github.com/ros2/rosidl/tree/master/rosidl_typesupport_introspection_c) in micro-ROS Galactic distribution, an automated memory handling for micro-ROS types is available. The tools related to this feature are available in the package [`micro_ros_utilities`](https://github.com/micro-ROS/micro_ros_utilities).
+
+The documentation of the package [`micro_ros_utilities`](https://github.com/micro-ROS/micro_ros_utilities) are available [here](https://micro.ros.org/docs/api/utils/).
```
Comment (Acuadros95) [inline _docs/tutorials/advanced/handling_type_memory/index.md:209]:
```suggestion
The documentation of the package [`micro_ros_utilities`](https://github.com/micro-ROS/micro_ros_utilities) is available [here](https://micro.ros.org/docs/api/utils/).
```

File: _docs/tutorials/advanced/handling_type_memory/index.md [hunk 2]
```diff
@@ -191,3 +201,122 @@ for(int32_t i = 0; i < 3; i++){
...
+conf.max_basic_type_sequence_capacity = 5;
+
+bool success = micro_ros_utilities_create_message_memory(
+  ROSIDL_GET_MSG_TYPE_SUPPORT(mypackage, msg, MyType),
+  &mymsg,
+  conf
+);
+```
+
+A complex rules approach can be used as in the following example:
```
Comment (Acuadros95) [inline _docs/tutorials/advanced/handling_type_memory/index.md:248]:
```suggestion
To customize the length of each member of the struct, a complex rules approach can be used as in the following example:
```

File: _docs/tutorials/advanced/handling_type_memory/index.md [hunk 3]
```diff
@@ -191,3 +201,122 @@ for(int32_t i = 0; i < 3; i++){
...
+
+micro_ros_utilities_memory_rule_t rules[] = {
+  {"multiheaders", 4},
+  {"multiheaders.frame_id", 60},
+  {"name", 10}
+};
+conf.rules = rules;
+conf.n_rules = sizeof(rules) / sizeof(rules[0]);
+
+// member values of MyComplexType will have the default max_basic_type_sequence_capacity
```
Comment (Acuadros95) [inline _docs/tutorials/advanced/handling_type_memory/index.md:263]: But this example is about setting the capacity of some of the members, no?

File: _docs/tutorials/advanced/handling_type_memory/index.md [hunk 4]
```diff
@@ -191,3 +201,122 @@ for(int32_t i = 0; i < 3; i++){
...
+
+Finally, a destruction function is also provided for messages allocated in dynamic memory:
+
+
+```c
+mypackage__msg__MyComplexType mymsg;
+
+// Memory allocation using micro_ros_utilities_create_message_memory
+// ...
+
```
Comment (Acuadros95) [inline _docs/tutorials/advanced/handling_type_memory/index.md:316]:
```suggestion
// Release memory previously allocated with micro_ros_utilities_create_message_memory

```

File: _docs/tutorials/advanced/handling_type_memory/index.md (hunk 3 above)
Comment (pablogs9) [reply _docs/tutorials/advanced/handling_type_memory/index.md:263]: Yes, we are using rules to set the size of some members and the other members such as `values` has default capacity. Maybe:

```suggestion
// member named "values" of MyComplexType will have the default max_basic_type_sequence_capacity
```

**Review by Acuadros95 — APPROVED** [review]

**Pull request merged**